app/views/issues/merging.html.erb

```diff
@@ -0,0 +1,56 @@
+<% content_for :title, 'Merge issues' %>
+
+<% content_for :sidebar do %>
+  <%= render 'sidebar'%>
+<% end %>
+
+<div id="issues_editor">
+  <div class="inner note-text-inner">
+    <h3>Merging <%= @issues.count %> issues</h3>
+
+    <%= simple_form_for Issue.new, url: merge_issues_path, method: 'post' do |f| %>
+      <p class="text-warning">WARNING: Only the target issue will prevail, the others will be deleted.</mark>
+      <p>All issues evidences will be moved to the selected issue.
+      <div class="row-fluid">
+        <div class="span6">
+          <p class="merge_option">Select existing issue to merge the others into.
+          <p class="merge_option" style="display: none">Merge issues into a new issue.
+        </div>
+        <div id="change_merge_option" class="span6 pull-right">
+          <a class="merge_option" href="#">Or click here to merge into a new issue</a>
+          <a class="merge_option" style="display: none" href="#">Or click here to merge into a existing issue</a>
```

Comment:
This text still isn't really clear enough, especially the WARNING in the first sentence (it sounds weird using the word "prevail" to talk about data).

I would rephrase the whole thing and give more detail, and change the order slightly:

> ## Merging X Issues
> 
> Select a target issue to merge the other issues into.
> 
> All issues except the target issue will be deleted, and their evidence will be moved to the target issue.
>
> WARNING: The text of all issues except the target will be lost. If you want to preserve the text, [click here to merge all issues into a new issue with custom text](#).

Then, when they click the link, change to:

> ## Merging X Issues
> 
> All issues will be deleted, and their evidence will be moved into a brand new issue.
>
> WARNING: Only the new text that you enter here will be saved; existing text in the old issues will be deleted.  [Click here to use one issue as a target issue that does not get deleted](#).

This wording might not be perfect, but I think it's worth being as explicit as possible about what happens, to avoid confusion. I also think it helps to describe exactly what happens in terms of how issues get deleted, because then users should be less confused when their "merged" issues appear in the Trash.